src/foam/comics/DAOControllerView.js

```diff
@@ -185,7 +190,10 @@ foam.CLASS({
                   this
                     .start()
                       .add(self.cls.PREDICATE.clone().copyFrom({
-                        view: { class: 'foam.u2.view.SimpleSearch' }
+                        view: { 
+                          class: 'foam.u2.view.SimpleSearch',
+                          searchValue: self.memento && self.memento.paramsObj.search ? self.memento.paramsObj.search : null
```

Comment:
Is this the the same as just: self.memento && self.memento.paramsObj.search,
except it gives an undefined instead of null in the bad case, but if you need the null specifically, you could do:
self.memento && self.memento.paramsObj.search || null